com.github.actionless.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "com.github.actionless.oomox",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "oomox-gui",
+    "finish-args": [
+        "--share=network",
```

Comment:
Does this actually use network access?

Reply:
without that i see `X: no display` and so on

Reply:
That would be strange and probably a `flatpak` bug...